qiskit_ibm_runtime/__init__.py

```diff
@@ -218,14 +216,9 @@
 
 from .estimator import (  # pylint: disable=reimported
     EstimatorV2,
-    EstimatorV1,
-    EstimatorV1 as Estimator,
-)
-from .sampler import (  # pylint: disable=reimported
-    SamplerV2,
-    SamplerV1,
-    SamplerV1 as Sampler,
+    EstimatorV2 as Estimator,
 )
+from .sampler import SamplerV2, SamplerV2 as Sampler  # pylint: disable=reimported
 from .options import Options, EstimatorOptions, SamplerOptions, OptionsV2
```

Comment:
There are a number of options class, such as `Options` and `TranspilationOptions` that are V1 only.